bayesian/api_v1.py

```diff
@@ -472,6 +485,11 @@ class StackAnalysesV2(ResourceWithSchema):
 
     @staticmethod
     def post():
+        decoded = decode_token()
+        if decoded == None:
```

Comment:
@fridex good catch. Actually, this check is present in login_required. Need to check why that did not work.